packages/flutter/test/material/text_field_focus_test.dart

```diff
@@ -548,6 +563,63 @@ void main() {
     expect(focusNodeA.hasFocus, false);
     expect(focusNodeB.hasFocus, true);
   }, variant: TargetPlatformVariant.desktop());
+
+  testWidgets('TextFormField gets blurred when tapping outside of the widget on mobile browsers', (WidgetTester tester) async {
+    final FocusNode focusNodeA = FocusNode();
```

Comment:
Above this line add: "Regression test for https://github.com/flutter/flutter/issues/64245"

Reply:
I've done it.